README.md

```diff
@@ -3,11 +3,11 @@
 [![codecov](https://codecov.io/gh/kubesphere/ks-devops/branch/master/graph/badge.svg?token=XS8g2CjdNL)](https://codecov.io/gh/kubesphere/ks-devops)
 [![Contributors](https://img.shields.io/github/contributors/kubesphere/ks-devops.svg)](https://github.com/kubesphere/ks-devops/graphs/contributors)
 
-KubeSphere DevOps integrates popular CI/CD tools, provides CI/CD Pipelines based on Jenkins, offers automation toolkits 
+KubeSphere DevOps integrates popular CI/CD tools, provides CI/CD pipelines based on Jenkins, offers automation toolkits 
```

Comment:
Jenkins community is using `Pipeline` instead of `pipeline`. See the following screenshot:

![image](https://user-images.githubusercontent.com/1450685/140917369-08fb8c4d-bdd2-422e-93eb-f3079cafdc68.png)

https://www.jenkins.io/doc/

Reply:
I see. Modified in[ this commit](https://github.com/kubesphere/ks-devops/pull/346/commits/c825ce342006b4d7afb5dd850b45edf863c04367).